pyproject.toml

```diff
@@ -15,6 +15,7 @@ requests = "2.31.*"
 django = "4.2.*" # should it be a dependency of the extension ? it is already a dependency of the sdk.
 
 # Remove when devmock is not needed anymore
+azure-monitor-opentelemetry-exporter = "^1.0.0b21"
```

Comment:
Why this is a dependency of the extension ?

Reply:
Agree, auto added